python_modules/libraries/dagster-databricks/dagster_databricks/components/databricks_workspace/schema.py

```diff
@@ -0,0 +1,56 @@
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Annotated, Any, Optional, Union
+
+from dagster.components import Resolver
+from pydantic import BaseModel
+
+from dagster_databricks.components.databricks_asset_bundle.configs import DatabricksJob
+
+DatabricksJobInfo = Union[dict[str, Any], DatabricksJob]
+
+
+@dataclass
+class DatabricksFilter:
+    include_job: Callable[[DatabricksJobInfo], bool]
+
+
+class IncludeJobsConfig(BaseModel):
+    job_ids: list[int]
+
+
+class DatabricksFilterConfig(BaseModel):
+    include_jobs: Optional[IncludeJobsConfig] = None
+
+
+def resolve_databricks_filter(context, config: DatabricksFilterConfig) -> DatabricksFilter:
+    """Convert a DatabricksFilterConfig into a DatabricksFilter."""
+    if config and config.include_jobs and getattr(config.include_jobs, "job_ids", None):
+        allowed_ids = set(config.include_jobs.job_ids)
+
+        def include_job(job: DatabricksJobInfo) -> bool:
+            job_id = None
```

Comment:
given that `job` will always be an instance of DatabricksJobInfo, you can just do `return job.job_id in allowed_ids`